src/werkzeug/local.py

```diff
@@ -5,11 +5,22 @@
 
 # Each thread has its own greenlet, use that as the identifier for the
 # context. If greenlets are not available fall back to the current
-# thread ident.
+# thread ident. Asyncio Task ident is preferred.
+
 try:
-    from greenlet import getcurrent as get_ident
-except ImportError:
-    from threading import get_ident
+    from asyncio import get_running_loop, current_task
+
+    get_running_loop()
```

Comment:
Why is this call needed? What if there's not an event loop at import time, but there is in the context where the local is being used?

What if this does succeed, but the application isn't using asyncio, they're using threads or gevent?